### DISCUSSION TITLE
Encrypting Laravel Eloquent models with CipherSweet | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#502] [General]
# Encrypting Laravel Eloquent models with CipherSweet | Rias.be

In your project, you might store some sensitive personal or credential data in your database. Should an unauthorised person get access to your DB, all this sensitive can be read.. To solve this problem, you can encrypt the data. This way, unauthorised persons cannot read it, but your application can...

https://rias.be/blog/encrypting-laravel-eloquent-models-with-ciphersweet?utm_source=newsletter&utm_medium=email&utm_campaign=freekdev-newsletter-153-corrected-version

**Comment by sanderdewijs** [comment]
In the past I used another package for encrypting values in the database. That package didn't play nice with Fortify however. Fortify did not use the method to search decrypted email adresses in the database. Does the package handle this in some way?

**Reply by riasvdv** [reply]
No, this package doesn't provide any integration with Fortify specifically, but you can use this https://laravel.com/docs/9.x/fortify#customizing-user-authentication to customize how users are fetched with Fortify
